integration_tests/test_mempool.py

```diff
@@ -61,24 +62,24 @@ def test_mempool(cronos_mempool):
     print(f"block number start: {block_num_0}")
     nonce_begin = w3.eth.get_transaction_count(address_from)
 
-    sended_hash_set = set()
+    txs_to_send = []
     for i in range(5):
         nonce = nonce_begin + i
-        tx = {
+        txs_to_send.append({
```

Comment:
but to do it in batch, it's not really a mempool test case anymore, if it was not stable enough, maybe try add a little bit sleep between the loops?

Reply:
strange that after 10 blocks still can't find `0x1127bfa542258dea57bb7a6b8cda1041a547097cb3e601d8e481c54a6f815907`, not sure it's confirmed or fail to included
```block number start: 5
all send tx hash: f{HexBytes('0xce12a45ee6e6a3112bf469dc90361106737b65f61af9168d00ae2b3856d74b01'), HexBytes('0x5ed67709eafb00d074d64b015dc27836e593271d0789d22e4b356a69498b3d77'), HexBytes('0x1127bfa542258dea57bb7a6b8cda1041a547097cb3e601d8e481c54a6f815907'), HexBytes('0x1abec2504f8dbdd4d4c9359f9e9810ee44c4c9ab4848fb1d6ba2ed17dcc6feb7'), HexBytes('0xbf28ee609a7758c0592fcf6c86f0d57da55f8b3bb8c383fba0defc99ea44cc0b')}
all pending tx hash after 1 block: [HexBytes('0xbf28ee609a7758c0592fcf6c86f0d57da55f8b3bb8c383fba0defc99ea44cc0b'), HexBytes('0xce12a45ee6e6a3112bf469dc90361106737b65f61af9168d00ae2b3856d74b01'), HexBytes('0x1abec2504f8dbdd4d4c9359f9e9810ee44c4c9ab4848fb1d6ba2ed17dcc6feb7'), HexBytes('0x5ed67709eafb00d074d64b015dc27836e593271d0789d22e4b356a69498b3d77')]
all pending tx hash after 2 block: []
all pending tx hash after 3 block: []
all pending tx hash after 4 block: []
all pending tx hash after 5 block: []
all pending tx hash after 6 block: []
all pending tx hash after 7 block: []
all pending tx hash after 8 block: []
all pending tx hash after 9 block: []
all pending tx hash after 10 block: []
```